src/main/java/com/github/lunatrius/schematica/client/util/BlockList.java

```diff
@@ -82,6 +83,7 @@ private WrappedItemStack findOrCreateWrappedItemStackFor(final List<WrappedItemS
         public ItemStack itemStack;
         public int placed;
         public int total;
+        public int hold; // cache current hold item counts
```

Comment:
`inventory` is more fitting, `hold` implies it uses only the "active" item stack.